### PR TITLE
sound page redesign

### LINKED ISSUE #111: ui redesign: update description's design
The description on dark mode no longer uses a primary color background.

## 🙅‍♀️ Bad (current implementation)

**Light Mode**
Screen Shot 2020-10-21 at 9 07 34 AM

**Dark Mode**
Screen Shot 2020-10-21 at 9 08 01 AM

## 👍 Good (new design)

**Light Mode**
Screen Shot 2020-10-21 at 9 06 50 AM
**Dark Mode**
Screen Shot 2020-10-21 at 9 03 33 AM

### PR DESCRIPTION
**Pull request by akshara-sun**
This PR will be showcasing changes made to the overall sound page. So far, CSS Styles have been added to the borders of the sound list, the download, edit, and play buttons.

Issue Tickets:
Closes #111, #125, #94 

To-do:
- [ ] Style buttons
    - [x] play/pause button below waveform 
    - [x] download button below waveform 
    - [x] edit button below waveform 
    - [ ] Similar and Pack List buttons 
- [x] Fix description padding 
- [ ] Format sound list icons
- [ ] Change description backgrounds to match Figma's dark theme.
- [ ] Convert CSS for buttons to Tailwind CSS

### CODE REVIEW

**Comment by akshara-sun** [comment]
**Current style:**
Screen Shot 2020-11-04 at 5 07 08 PM

Figma Version:
Screen Shot 2020-11-04 at 5 08 37 PM

File: src/styles.css [hunk 1]
```diff
@@ -77,3 +76,58 @@ h1, h1, h3, h4, h5, h6 {
   color: var(--primary-color);
   background: var(--secondary-color);
 }
+
+.download-container {
```
Comment (amilajack) [inline src/styles.css:80]: We can get rid of a lot of this css by using tailwind classes. Let me know if you need help with that.

**Review by amilajack — CHANGES_REQUESTED** [review]

**Comment by amilajack** [comment]
Please add a description to this PR. Currently there's no description, which makes it hard to review:

Screen Shot 2020-11-05 at 8 59 10 AM

File: src/styles.css (hunk 1 above)
Comment (akshara-sun) [reply src/styles.css:80]: > Please add a description to this PR. Currently there's no description, which makes it hard to review:
> 
> Screen Shot 2020-11-05 at 8 59 10 AM

@amilajack @hamirmahal Added description to PR, issue ticket info, and To-do's.

Comment (akshara-sun) [reply src/styles.css:80]: > We can get rid of a lot of this css by using tailwind classes. Let me know if you need help with that.

For sure, will let you know if I need any help.
Thank you.

**Comment by akshara-sun** [comment]
Description Padding [BEFORE]
Screen Shot 2020-11-07 at 11 12 42 AM

Description Padding [NOW]
Screen Shot 2020-11-07 at 11 13 13 AM

**Comment by amilajack** [comment]
Please update the PR description to indicate which things were fixed so far. Use checkboxes

**Review by amilajack — APPROVED** [review]

**Pull request merged**